.github/issue_template.md

```diff
@@ -0,0 +1,17 @@
+### Steps to reproduce
+
+Which steps led to the issue?
+
+### Expected behaviour 
+
+What should have happened?
+
+### Actual behaviour
+
+What actually happened?
+
+### System configuration
+
+* Operating System
```

Comment:
This could be a prepared markdown table.
Add used drupal version to the questions

Reply:
Thank you for your feedback and do excuse the delayed response. Your suggestions have been integrated.